eng/pipelines/coreclr/superpmi-asmdiffs-checked-release.yml

```diff
@@ -0,0 +1,41 @@
+trigger:
+  batch: false
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - src/coreclr/jit/*
+    - src/coreclr/inc/jiteeversionguid.h
```

Comment:
Don't need this 'jiteeversionguid.h" line: there's no point attempting asm diffs if the interface changed because the existing spmi collection won't work for replay and the new one won't be available.

Reply:
Removed.